# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

move_baseActionGoal action_goal
move_baseActionResult action_result
move_baseActionFeedback action_feedback
